docs/static/releasenotes.asciidoc

```diff
@@ -23,6 +24,27 @@ This section summarizes the changes in the following releases:
 * <<logstash-8-0-0-alpha2,Logstash 8.0.0-alpha2>>
 * <<logstash-8-0-0-alpha1,Logstash 8.0.0-alpha1>>
 
+
+[[logstash-8-4-0]]
+=== Logstash 8.4.0 Release Notes
+
+coming[8.4.0]
+
+[[features-8.4.0]]
+==== New features and enhancements
+
+
+[[notable-8.4.0]]
+==== Performance improvements and notable issues fixed
+
+
+[[core-8.4.0]]
+==== Other changes to Logstash core
```

Comment:
As usual, we may not need all of these headings.  I added a section for "other changes" to hold items that don't fit neatly into new features/enhancement or notable issues. If we don't need it the section, we can delete it.